app/mailers/signup_followup_mailer.rb

```diff
@@ -1,5 +1,8 @@
 class SignupFollowupMailer < ApplicationMailer
-  def followup(email_address)
+  default from: Rails.configuration.address_for_transactional_authentication_emails
```

Comment:
@alaroia-cfa How do you feel about us updating this "transactional authentication" emails configuration to just "transactional" emails? I think now we're using it in multiple other places where we need a no-reply email, and im not sure that there's a distinct reason to make the designation

Reply:
Big +1 - let's rename it to `transactional` something or `noreply` something in the config